pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -182,15 +182,15 @@ def get_artifact(self, obj):
         """
         Get atrifact summary.
         """
-        return ArtifactRefSerializer(self.context["content_artifact"]).data
+        return ArtifactRefSerializer(obj.contentartifact_set.first()).data
```

Comment:
Since this is replacing the removed line `artifact = ContentArtifact.objects.get(content=instance)` I think this could be 
```suggestion
        return ArtifactRefSerializer(obj.contentartifact_set.get()).data
```